.travis.yml

```diff
@@ -1,10 +1,10 @@
 sudo: false
 language: node_js
 node_js:
-  - "6"
```

Comment:
Note that I dropped Node 6 because it's reached end of life: https://nodejs.org/en/about/releases/